lib/webpacker/configuration.rb

```diff
@@ -36,6 +36,10 @@ def source
       fetch(:source_path)
     end
 
+    def compile?
+      fetch(:compile).nil? ? Webpacker.env.test? : fetch(:compile)
```

Comment:
@rossta Yes please 👍  add it to Chagelog and highlight if it's a breaking change